src/main/res/values/strings.xml

```diff
@@ -160,38 +160,38 @@
     <string name="unstable_updates">Unstable updates</string>
     <string name="unstable_updates_summary">Suggest installing unstable versions</string>
     <string name="unverified">Unverified</string>
-    <string name="update">Update</string>
-    <string name="updates">Updates</string>
-    <string name="upstream_source_code_is_not_free">The upstream source code is not free</string>
+    <string name="update">Upgrade</string>
+    <string name="updates">Upgrades</string>
```

Comment:
Updates != Upgrades and in this context, it's still updates and not per se upgrades

Reply:
What is the context? I think it might be possible to avoid those words altogether.

Reply:
Will revert and hope to avoid sometime in the future.
There is no knowing what this is when just translating without context.